src/fixturelibrary.ts

```diff
@@ -127,6 +149,25 @@ export class FixtureLibrary {
     return fixture;
   }
 
+  public async getMakeModelIndex(): Promise<MakeModel[]> {
```

Comment:
This seems like it would be a very expensive function to run. I think this will load the whole fixture library into memory and perform a lot of disk io for every fixture?

That sounds like it would be expensive for autocomplete, why not add a `make` and `model` to `index.json`?

Calculate it once at build time, then we already have everything we need?